From 43afba125475532cd2ae587c51902dd669769020 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Sun, 30 Oct 2016 14:00:40 +0200
Subject: [PATCH 1032/2241] Documentation: dt-bindings: Add a new compatible
 for cp110 ahci

A8k requires workaround for SATA in order to be functional on
both A0 and A1 (due to incorrect port register offset).
A3700 doesn't require this workaround so we need to be able to
distinguish  a8k from A3700.

Change-Id: Iae0bacb5bdc0503b32429a267464145656775678
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33467
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Omri Itach <omrii@marvell.com>
---
 Documentation/devicetree/bindings/ata/ahci-platform.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/ata/ahci-platform.txt b/Documentation/devicetree/bindings/ata/ahci-platform.txt
index b3983e0..d1f9225 100644
--- a/Documentation/devicetree/bindings/ata/ahci-platform.txt
+++ b/Documentation/devicetree/bindings/ata/ahci-platform.txt
@@ -14,6 +14,7 @@ Required properties:
   - "ibm,476gtr-ahci"
   - "marvell,armada-380-ahci"
   - "marvell,armada-3700-ahci"
+  - "marvell,armada-cp110-ahci"
   - "snps,dwc-ahci"
   - "snps,exynos5440-ahci"
   - "snps,spear-ahci"
-- 
2.7.4

